hack/provision.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# DESCRIPTION:
+# This is a script that bootstraps basic content in the OpenShift server
+# to facilitate development flows and users that want to kick the tires.
+# 
+# PROVISIONS:
+# * Users
+# ** test-cluster-admin
+# ** test-admin
+# * Docker registry
+# * OpenShift router
+# * OpenShift templates
+
+# INSTRUCTIONS:
+# $ git clone ...
+# $ make all
+# $ openshift start
+# $ hack/provision.sh
+
+# Export necessary environment variables for resources created on openshift start
+export ADMIN_KUBECONFIG=`pwd`/openshift.local.config/master/admin.kubeconfig
+export REGISTRY_KUBECONFIG=`pwd`/openshift.local.config/master/openshift-registry.kubeconfig
+export ROUTER_KUBECONFIG=`pwd`/openshift.local.config/master/openshift-router.kubeconfig
+export CA=`pwd`/openshift.local.config/master
+
+# Modify permissions on each config
+sudo chmod -R a+rwx "$CA"
+sudo chmod a+rwX "$ADMIN_KUBECONFIG"
+sudo chmod a+rwX "$REGISTRY_KUBECONFIG"
+sudo chmod a+rwX "$ROUTER_KUBECONFIG"
+
+# Run as admin
+export KUBECONFIG=$ADMIN_KUBECONFIG
```

Comment:
debating if it's better to just always pass --config instead of setting this.  i sometimes source scripts instead of running them, and now i'd end up with KUBECONFIG set in my shell.  but maybe i'm alone in that.